### PR TITLE
Ref frame debug

### LINKED ISSUE #7: ReferenceFrame not properly set through config.txt
Whether ReferenceFrame is set to 0 or 1 in the config.txt file, CyRSoXS  runs the default setting (0). This only affects command line usage. ReferenceFrame gets set properly through the pybind interface.

### PR DESCRIPTION
**Pull request by pdudenas**
This pull request fixes issue #7.

### CODE REVIEW

File: src/cudaMain.cu [hunk 1]
```diff
@@ -1338,8 +1338,8 @@ int cudaMainStreams(const UINT *voxel,
             START_TIMER(TIMERS::IMAGE_ROTATION)
           }
 #endif
-          const double alpha = cos(-Eangle);
-          const double beta = sin(-Eangle);
+          const double alpha = cos(Eangle);
+          const double beta = sin(Eangle);
```
Comment (pbeaucage) [inline src/cudaMain.cu:1342]: How have we tested that this sign flip is correct?

File: include/Input/InputData.h [hunk 2]
```diff
@@ -298,6 +298,10 @@ class InputData {
     if(ReadValue(cfg,"ScatterApproach",scatterApproach)){}
     if(ReadValue(cfg,"DumpMorphology",dumpMorphology)){}
     if(ReadValue(cfg,"MaxStreams",numMaxStreams)){}
+    int _temp1;
+    if(ReadValue(cfg,"ReferenceFrame",_temp1)){
+       referenceFrame = static_cast<bool>(_temp1);
+    }
```
Comment (pbeaucage) [inline include/Input/InputData.h]: Should we print a warning here if the user specifies material reference frame, basically just something like "the accuracy of material reference frame simulations is currently being investigated, this mode should not be used in production, for details see https://www.github.com/usnistgov/cyrsoxs/issues/7"?

**Review by pbeaucage — COMMENTED** [review]
See notes above.  Technically changing the default reference frame is an API breaking change but I think it's probably OK.  I am tempted to suggest that we warn that the default has changed if the user does not supply an explicit reference frame, but there's a delicate balance around swamping the user.

File: src/cudaMain.cu (hunk 1 above)
Comment (KumarSaurabh1992) [reply src/cudaMain.cu:1342]: This was a bug that somehow got missed out. The current one is correct. The negative sign should not be there.

File: include/Input/InputData.h (hunk 2 above)
Comment (KumarSaurabh1992) [reply include/Input/InputData.h]: I think that is a good idea. I can add the warning.

**Comment by KumarSaurabh1992** [comment]
The commit https://github.com/usnistgov/cyrsoxs/pull/9/commits/4e8dce740d2587ddc5c4c8adbda6511d22d7cc7e fixes two issues:

1. Checks if the user has input some random input to reference frame. Must only be 0/1
2. Also, raise the warning if the Material is set.

File: src/cudaMain.cu (hunk 1 above)
Comment (pdudenas) [reply src/cudaMain.cu:1342]: Yes, tested to be correct and match algorithm 1.

Comment (pbeaucage) [reply src/cudaMain.cu:1342]: I am adding a followup issue for this as #10 so we don't lose the thread of adding a unit test/functional test.

**Review by pbeaucage — APPROVED** [review]
LGTM.  Thanks @KumarSaurabh1992 and @pdudenas!

**Pull request merged**